rustls-post-quantum/src/lib.rs

```diff
@@ -59,12 +61,21 @@ use rustls::crypto::{
 use rustls::ffdhe_groups::FfdheGroup;
 use rustls::{Error, NamedGroup, PeerMisbehaved, ProtocolVersion};
 
-/// A `CryptoProvider` which includes `X25519MLKEM768` key exchange.
+mod mlkem;
```

Comment:
I thought our pref'd ordering was `mod` -> `pub use` -> `use` but I don't think it's something made explicit in the [ordering](https://github.com/rustls/rustls/blob/main/CONTRIBUTING.md#ordering) section of the contrib guide.

If my gut check on the expected order is correct I can add some text about it. WDYT?

Reply:
Not sure what you're objecting too here? This order looks good to me, although I would not have the empty line between the `mod` declaration and any imports from that module.

I think my preferred order is (1) imports from other crates and other modules, (2) module declarations, (3) keep imports from locally-declared modules near their declarations.

Reply:
> Not sure what you're objecting too here?

This is ordering `use` imports, then declares a `mod`, then adds `pub use` for items from the mod. As mentioned I thought it should be declare `mod`, export items from mod `pub use`, import stuff with `use`.

> I think my preferred order is (1) imports from other crates and other modules, (2) module declarations, (3) keep imports from locally-declared modules near their declarations.

@ctz Is that your preference too? If so I think it should be put into CONTRIBUTING.md.

I don't feel strongly except that it shouldn't be ambiguous per-PR.

Reply:
I don't have a strong opinion in this area. It kind of makes sense to me that the order would go 1) private `use`s, 2) `mod`s, 3) `pub use`s. In practice I think the number of files with all three should be pretty low -- I'd prefer if files that have a ton of `mod`s don't also have non-trivial code in.

Reply:
I'll make a CONTRIB update for this. Thanks!